overlord/devicestate/devicemgr.go

```diff
@@ -479,6 +482,47 @@ func (m *DeviceManager) ensureBootOk() error {
 	return nil
 }
 
+func (m *DeviceManager) ensureInstalled() error {
```

Comment:
A someone who participates in core20 only partially I'd love to see a comment that explains how this function fits in the overall idea. It's much easier to follow such comments than to re-create the impression from the code.

Reply:
The install and boot documentation is still incomplete with some details yet to be defined. The basic idea here is that the core20 image is shipped with the seed partition only, and must be properly installed on the target system. To proceed with the installation, the system boots in ephemeral mode, creates the remaining partitions that are specified in gadget, and prepares the boot environment for run mode. The actual install handler is part of a different PR.

Reply:
But of course we can comment this better once we have the whole process defined.